src/icons.jsx

```diff
@@ -14,6 +14,7 @@ const importIcons = function () {
     'icon-check-circleless',
     'icon-clock',
     'icon-cozy',
+    'icon-cozy-dark',
```

Comment:
May I suggest to call this icon `icon-cozy-full` or something like that?
`Dark` suggest a sense of tone which is not really the case. You could also call it `icon-cozy-negative` if you will, just something not "color" related.

Reply:
I was referring to dark themes, but ok for `negative`

Reply:
Sure but it could be misunderstood by others :)